dataset/include/scipp/dataset/map_view.h

```diff
@@ -184,7 +181,4 @@ template <class Masks>
 SCIPP_DATASET_EXPORT Variable masks_merge_if_contained(const Masks &masks,
                                                        const Dimensions &dims);
 
-[[nodiscard]] SCIPP_DATASET_EXPORT Coords copy(const Coords &coords);
-[[nodiscard]] SCIPP_DATASET_EXPORT Masks copy(const Masks &masks);
```

Comment:
Why removed?

Reply:
They are declared in https://github.com/scipp/scipp/blob/main/dataset/include/scipp/dataset/map_view_forward.h So the declarations here are redundant.